src/xFactor/controller/StageController.java

```diff
@@ -0,0 +1,148 @@
+package xFactor.controller;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import xFactor.base.Application;
+import xFactor.infrastructure.model.JudgeFavourite;
+import xFactor.infrastructure.model.Participant;
+import xFactor.infrastructure.model.Stage;
+
+@RestController
+public class StageController {
+	@Autowired
+	Application app;
+
+	@RequestMapping(value = "/number-of-stages", method = RequestMethod.GET, produces = "application/json")
+	public @ResponseBody int getNumberOfStages() {
+		app.setNumberOfStages(3);
+		return app.getNumberOfStages();
+	}
+
+	Stage stage1;

```

Comment:
you should not do this. Whenever you need a stage, you will have to pass it's ID as a parameter in the API and fetch it from there